pages/api/conversations/index.js

```diff
@@ -0,0 +1,74 @@
+import createEndpoint from 'lib/api/utils/createEndpoint';
+import Response from 'lib/api/domain/Response';
+import { createConversation } from 'lib/dependencies';
+import { Conversation } from '../../../lib/domain';
+
+export const endpoint = ({ createConversation }) =>
+  createEndpoint(
+    {
+      allowedMethods: ['POST'],
+      validators: [
+        {
+          name: 'firstName',
+          failureMessage: 'first name is required',
+          validate: ({ body }) => body.firstName && body.firstName.length > 0
+        },
+        {
+          name: 'lastName',
+          failureMessage: 'last name is required',
+          validate: ({ body }) => body.lastName && body.lastName.length > 0
+        },
+        {
+          name: 'email',
+          failureMessage: 'resident email is required',
+          validate: ({ body }) => body.email && body.email.length > 0
+        },
```

Comment:
Ln23 failureMessage specifies that email field belongs to resident. Should we do the same for firstName & lastName?